src/types.ts

```diff
@@ -345,6 +345,23 @@ type TradersHubOnboardingFormAction = {
 type UpgradeMT5BannerAction = {
     action: 'open' | 'push_cta_upgrade'
 }
+type TradersHubPerformanceMetricsAction = {
+    action:
+        | 'create_mt5_account_time'
```

Comment:
@sergei-deriv how we can differentiate events for legacy TradersHub and TradersHub new repo in the dashboard while they are both using same actions?